### DISCUSSION TITLE
Truncated file names on the release page makes it almost impossible to pick the right release, see screenshot

### DISCUSSION THREAD

**Discussion by 1110101** [community/community#67897] [Mobile] [answered]
![Screenshot_20230921-231135_GitHub.png](https://github.com/community/community/assets/6941073/ff316c04-de7d-4785-a1f4-439d940ad013)

**Comment by stevepopovich** [comment, marked as answer]
Hi @1110101 this is fair feedback. I will pass it on to the team for consideration. Thanks for writing in!

**Reply by 1110101** [reply]
Awesome, thank you!

**Comment by SumanS7865** [comment]
Response #1

**Comment by maxgtr992** [comment]
Hi @1110101 this is fair feedback. I will pass it on to the team for consideration. Thanks for writing in!
